Header header

uint32 INVALID=0
uint32 CONVERGING=1
uint32 GOOD=2

uint32 type
